app/assets/stylesheets/articles.scss

```diff
@@ -1,3 +1,71 @@
 // Place all the styles related to the Articles controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import "colours";
+
+h1 {
+  text-align: center;
+}
+
+textarea {
+  width: 90%;
+  height: 150px;
+}
+
+.centre {
+  width: 50%;
+  margin: 0 auto;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+}
+
+.article-menu {
+  display: flex;
+  flex-direction: column;
+}
+
+.article-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: lighten($accent, 50%);
+  margin-top: 5px;
+  margin-bottom: 5px;
+  padding: 6px;
+
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  a:visited {
+    color: black;
+  }
+
+  a:hover {
+    color: $accent;
+  }
+}
+
+.article-actions {
+  display: flex;
+  justify-content: space-around;
+}
+
+.action {
+  margin-right: 20px;
+}
+
+.entry-text {
```

Comment:
Elsewhere you refer to articles - would `article-text` be more clear/consistent?